### PR TITLE
All interface

### PR DESCRIPTION
**Pull request by Lmaxlon**
updated all interfaces, fixed token

### CODE REVIEW

File: app/interface_student.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+from app.states import get_user_state, set_user_state, get_user_data, set_user_data, clear_user_data
+from app.api.student import Student
+''', StudentJoinGroupException, StudentLeaveGroupException'''
+
+
+class StudentInterface:
+    def __init__(self, bot: Bot, student_name: str):
+        self.bot = bot
+        #self.database = Database() #добавить
+        self.student = Student(student_name)
```
Comment (ggerlakh) [inline app/interface_student.py]: Seems like you missed second 'database' parameter in 'Student' constructor https://github.com/six-nine/edu_queue/blob/main/app/api/student.py#L25

File: app/interface_student.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+    async def process_invite_code(self, message: types.Message, invite_code: str):
+        try:
+            invite_code_int = int(invite_code)
+            student_tg_id = message.from_user.id
+            result = self.student.join_group(invite_code_int, student_tg_id)
```
Comment (ggerlakh) [inline app/interface_student.py]: invite_code has type str in join_group method https://github.com/six-nine/edu_queue/blob/main/app/api/student.py#L31

File: app/interface_student.py (hunk 1 above)
Comment (Lmaxlon) [reply app/interface_student.py]: fixed

File: app/interface_student.py (hunk 2 above)
Comment (Lmaxlon) [reply app/interface_student.py]: fixed

File: app/db/database_config.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import sys
+from os import getenv
+#from dotenv import load_dotenv
+from db import Database
```
Comment (dee-tree) [inline app/db/database_config.py:4]: `from db.db import Database`

File: app/interface_educator.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
+from aiogram import Bot, types
+from app.states import get_user_state, set_user_state, get_user_data, set_user_data
```
Comment (dee-tree) [inline app/interface_educator.py:2]: looks like it should be: `from states import ...`

File: app/interface_student.py [hunk 5]
```diff
@@ -0,0 +1,165 @@
+from aiogram import Bot, types
+from app.states import get_user_state, set_user_state, get_user_data, set_user_data, clear_user_data
```
Comment (dee-tree) [inline app/interface_student.py:2]: `from states import ...`

File: app/interface_student.py [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+from app.api.student import Student
```
Comment (dee-tree) [inline app/interface_student.py:3]: `from api.student import Student`

File: app/interface_student.py [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+''', StudentJoinGroupException, StudentLeaveGroupException)'''
+from app.db.database_config import db
```
Comment (dee-tree) [inline app/interface_student.py:5]: `from db.database_config import db`

File: app/main.py [hunk 8]
```diff
@@ -0,0 +1,112 @@
+import asyncio
+import logging
+
+from aiogram import Bot, Dispatcher, Router, types
+from aiogram.filters import Command
+from aiogram.types import CallbackQuery
+
+from app.interface_student import StudentInterface
```
Comment (dee-tree) [inline app/main.py:8]: `from interface_student import StudentInterface`

File: app/main.py [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+from config import API_TOKEN
```
Comment (dee-tree) [inline app/main.py:9]: Probably config should be moved to /app folder to have imports working

File: app/main.py [hunk 10]
```diff
@@ -0,0 +1,112 @@
...
+from app.interface_educator import EducatorInterface
+from app.states import set_user_state, get_user_state, get_user_data, set_user_data, clear_user_data
```
Comment (dee-tree) [inline app/main.py:11]: `from states import ...`

File: app/main.py [hunk 11]
```diff
@@ -0,0 +1,112 @@
+import asyncio
+import logging
+
+from aiogram import Bot, Dispatcher, Router, types
+from aiogram.filters import Command
+from aiogram.types import CallbackQuery
+
+from app.interface_student import StudentInterface
+from config import API_TOKEN
+from app.interface_educator import EducatorInterface
```
Comment (dee-tree) [inline app/main.py:10]: `from interface_educator import EducatorInterface`

File: app/db/database_config.py (hunk 3 above)
Comment (Lmaxlon) [reply app/db/database_config.py:4]: fixed

File: app/interface_student.py (hunk 5 above)
Comment (Lmaxlon) [reply app/interface_student.py:2]: no, in the app folder there is states.py for states:

ser_states = {}
user_data = {}

def get_user_state(user_id):
    return user_states.get(user_id)

def set_user_state(user_id, state):
    user_states[user_id] = state

def get_user_data(user_id):
    return user_data.get(user_id, {})
...

File: app/interface_educator.py (hunk 4 above)
Comment (Lmaxlon) [reply app/interface_educator.py:2]: no, in the app folder there is states.py for states:

ser_states = {}
user_data = {}

def get_user_state(user_id):
return user_states.get(user_id)

def set_user_state(user_id, state):
user_states[user_id] = state

def get_user_data(user_id):
return user_data.get(user_id, {})
...

File: app/interface_student.py (hunk 6 above)
Comment (Lmaxlon) [reply app/interface_student.py:3]: fixed

File: app/interface_student.py (hunk 7 above)
Comment (Lmaxlon) [reply app/interface_student.py:5]: fixed

File: app/main.py (hunk 8 above)
Comment (Lmaxlon) [reply app/main.py:8]: fixed

File: app/main.py (hunk 9 above)
Comment (Lmaxlon) [reply app/main.py:9]: fixed

File: app/main.py (hunk 11 above)
Comment (Lmaxlon) [reply app/main.py:10]: fixed

**Pull request merged**